src/TodoContainer.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-param-reassign */
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { Routes, Route } from 'react-router-dom';
+import TodoList from './components/TodoList';
+import Header from './components/Header';
+import InputTodo from './components/InputTodo';
+import About from './functionBased/About';
+import NotMatch from './functionBased/NotMatch';
+
+let ToDoArray = JSON.parse(localStorage.getItem('ToDoArray') || '[]');
+const componentDidMount = () => {
```

Comment:
Using the name "`componentDidMount`" for this function does not correspond to its purpose. You don't use this function in `useEffect` function which performs the role of a class component's`componentDidMount` function. Kindly change the name of the function.

Reply:
Please change the name of the function